### ISSUE TITLE
[Feature]: Experiment with alternative schedulers for `scheduler_g` and `scheduler_d`

### ISSUE THREAD

**Issue by Bebra777228** [IAHispano/Applio#945]
### Description

In the `train.py` file, there is a section for [initializing schedulers](https://github.com/IAHispano/Applio/blob/main/rvc/train/train.py#L489). Have you tried replacing the `ExponentialLR` class with a different scheduler, such as [CosineAnnealingLR](https://github.com/pytorch/pytorch/blob/main/torch/optim/lr_scheduler.py#L1046) or [ReduceLROnPlateau](https://github.com/pytorch/pytorch/blob/main/torch/optim/lr_scheduler.py#L1244), or any other scheduler?

I believe it would be beneficial to experiment with different scheduler classes. There might be a better option than `ExponentialLR` that could improve the training process.

### Problem

—

### Proposed Solution

—

### Alternatives Considered

—

**Comment by blaisewf** [comment]
“I believe it would be beneficial to experiment with different scheduler classes” Can you share any result of this? Not tested by us atm

**Comment by blaisewf** [comment]
could you try it an share your feelings?

**Comment by Bebra777228** [comment]
Sure, I'll give it a try 👌

**Comment by AznamirWoW** [comment]
AdamW may benefit from some warmup scheduler, RAdam does not need it.

**Comment by Bebra777228** [comment]
Overall, I didn't notice any significant improvements, so I think this code enhancement would be unnecessary. 🤷‍♂️
